doc/charm++/manual.rst

```diff
@@ -3852,7 +3852,7 @@ manner:
 .. code-block:: c++
 
      msg->arr1[13] = 1;
-     msg->arr2[5] = 32.82;
```

Comment:
Is this change intentional?

Reply:
Nope, I'll revert it. (I did this in GitHub's editor and I must have messed up when searching for "32".)